rkt/run.go

```diff
@@ -328,6 +340,22 @@ func (pl *portList) Type() string {
 	return "portList"
 }
 
+// flagStringList implements the flag.Value interface to contain a set of strings
+type flagStringList []string
```

Comment:
It's here - https://github.com/coreos/rkt/blob/master/tools/common/util.go#L24

Maybe we should move this thing elsewhere.